lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -144,9 +144,13 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     otherMetrics.forEach(item => {
       metricsColumn2El.appendChild(this._renderMetric(item));
     });
-    const estValuesEl = this.dom.createChildOf(metricsColumn2El, 'div',
-        'lh-metrics__disclaimer lh-metrics__disclaimer');
-    estValuesEl.textContent = Util.UIStrings.varianceDisclaimer;
+
+    // 'Values are estimated and may vary' is used as the category description for PSI
+    if (environment !== 'PSI') {
```

Comment:
Is this being covered by tsc right now? :)

Reply:
@patrickhulce whatcha mean

Reply:
meaning is the magic string  `'PSI'` being checked to make sure we're using it correctly vs. `psi`, typo `PS1` etc